README.md

```diff
@@ -65,160 +65,162 @@ Any contribution is welcome!
+* [Natron](https://natron.fr) - Open Source Compositing Software For VFX and Motion Graphics.
 * [OBS](https://github.com/obsproject/obs-studio) - software designed for capturing, compositing, encoding, recording, and streaming video content, efficiently.
-* [Olive](https://www.olivevideoeditor.org/) - Non-linear video editor aiming to provide a fully-featured alternative to high-end professional video editing software
+* [Olive](https://www.olivevideoeditor.org/) - Non-linear video editor aiming to provide a fully-featured alternative to high-end professional video editing software.
 * [OpenShot](https://github.com/OpenShot/openshot-qt) - award-winning free and open-source video editor for Linux, Mac, and Windows, and is dedicated to delivering high quality video editing and animation solutions to the world.
-* [qctools](http://bavc.github.io/qctools/) - A free and open source software tool that helps users analyze and understand their digitized video files through use of audiovisual analytics and filtering
-* [Shotcut](https://www.shotcutapp.com/) - A free, open source, cross-platform video editor
-* [tlRender](https://github.com/darbyjohnston/tlRender) - timeline render, is an early stage project for rendering editorial timelines
-* [UltraGrid](https://github.com/CESNET/UltraGrid) - UltraGrid low-latency audio and video network transmission system
-* [xSTUDIO](https://www.dneg.com/xstudio/) - Image playback and review for the filmmaking community
-* [RV](https://adsknews.autodesk.com/news/rv-open-source) - A digital review tool for film, TV, and games
+* [qctools](http://bavc.github.io/qctools/) - A free and open source software tool that helps users analyze and understand their digitized video files through use of audiovisual analytics and filtering.
+* [Shotcut](https://www.shotcutapp.com/) - A free, open source, cross-platform video editor.
+* [tlRender](https://github.com/darbyjohnston/tlRender) - timeline render, is an early stage project for rendering editorial timelines.
+* [UltraGrid](https://github.com/CESNET/UltraGrid) - UltraGrid low-latency audio and video network transmission system.
+* [xSTUDIO](https://www.dneg.com/xstudio/) - Image playback and review for the filmmaking community.
+* [RV](https://adsknews.autodesk.com/news/rv-open-source) - A digital review tool for film, TV, and games.
 
 ### UV/Unwrap library
 
 * [Thekla atlas](https://github.com/Thekla/thekla_atlas) | This tool performs mesh segmentation, surface parameterization, and chart packing.
 * [xatlas](https://github.com/jpcy/xatlas) | Fork from [theakla atlas](https://github.com/Thekla/thekla_atlas), it's a library to generate an UV for -example- lightmap uv.
 
 ### Tools
-
-*A set of small tools, DCC addons, python library... to complete or add feature on DCC softwares.*
+*A set of small tools, DCC addons, python library... to complete or add feature on DCC software.*
```

Comment:
Let the header breathe. :P

Keep a line break around top and bottom of this paragraph.